src/psyclone/core/single_variable_access_info.py

```diff
@@ -218,8 +218,8 @@ def is_written(self):
 
     def is_written_first(self):
         ''':returns: True if this variable is written in the first access
```

Comment:
This is missing line-continuation symbols I think (although the ref. guide isn't warning about that).

Reply:
The line-continuation symbol is actually not required, as long as you intend the next line correctly. But since PSyclone had the continuation lines from the very beginning, I never challenged that :)  I've added the missing \.

Reply:
Ah! I live and learn, thanks Joerg. In future then I suggest we do without them :-)